packages/acs-ui-javascript-loaders/src/utils.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Checks the version of react and throws an error if it is less than 18 in the loaders
+ * @param version - version of react
+ * @private
+ */
+export const parseReactVersion = (version: string | undefined): void => {
+  if (!version) {
+    return;
+  }
+  const reactVersion = version.split('.').map((v) => parseInt(v));
+  if (reactVersion[0] && reactVersion[0] < 18) {
+    throw new Error(
+      'React version is less than 18. Please upgrade to React 18 or alternatively checkout how to use our composites directly here: https://azure.github.io/communication-ui-library/?path=/docs/quickstarts-composites--page'
```

Comment:
We should probably get this behind an aka.ms url, for the upcoming storybook 8 switchover.

Reply:
are we going to reassign the storybook aka link to v8? or is there going to be a new aka link while we are vetting it?

Reply:
The link could be updated when we switch; so this code won't need updating at that time. If it's left this way, we'll need to keep note that this URL will need to updated potentially.